content/guide/property-system.md

```diff
@@ -0,0 +1,315 @@
+---
+title: Property
+---
+
+The property system in NativeScript is a way to expose a native property when you are exposing a native UI component.
```

Comment:
The property system in NativeScript allows for the exposure of native properties when exposing a native UI component.